app/controllers/admin/kudos_controller.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Admin
+  class KudosController < AdminController
+    before_action :set_kudo, only: %i[show edit update destroy]
```

Comment:
Actually nobody really uses callbacks, because they are making code harder to read - let's leave it like this -  you will see it by yourself after couple more actions added, and then decide to refactor it ;)

Reply:
Ok :)